core/src/raw/oio/buf/chunked_bytes.rs

```diff
@@ -74,9 +76,9 @@ impl ChunkedBytes {
     /// Reference: <https://doc.rust-lang.org/stable/std/collections/struct.VecDeque.html#impl-From%3CVec%3CT,+A%3E%3E-for-VecDeque%3CT,+A%3E>
     pub fn from_vec(bs: Vec<Bytes>) -> Self {
         Self {
+            size: bs.iter().map(|v| v.len()).sum(),
```

Comment:
I thought it was strange here.

Reply:
Can you explain a bit?

Reply:
I was referring to the part before the modification. I couldn't understand how the size of the buf, which was created from a certain bytes, could be 0.

Reply:
Yep, it's a bug.